scripts/create_custom_video_dataset.py

```diff
@@ -0,0 +1,105 @@
+            label = data[2]
+            
+            num_labels = len([file for file in os.listdir("{}/{}/".format(args.data_out_path, args.phase)) if int(label) == int(file.split("_")[0])])
+            label_folder = '{}_{}'.format(label, num_labels)
+            os.makedirs("{}/{}/{}".format(args.data_out_path, args.phase, label_folder))
+            label_anns['id'].append(label_folder)
+            label_anns['video_id'].append(label_folder)
+            label_anns['class'].append(label)
+            label_anns['start_frame'].append(frame_start)
+            label_anns['end_frame'].append(frame_end)
+
+            meta_anns['id'].append(label_folder)
+            meta_anns['video_id'].append(label_folder)
+            meta_anns['start_frame'].append(frame_start)
+            meta_anns['end_frame'].append(frame_end)
+
+            if args.from_video:
+                while frame_start > frame_count:
+                    retval, frame = cap.read()
+                    frame_count += 1
+                while frame_end > frame_count:
+                    retval, frame = cap.read()
+                    if retval:
+                        pass
+                        cv2.imwrite('{}/{}/{}/{}.jpg'.format(args.data_out_path, args.phase, label_folder, frame_count), frame)
+                    frame_count += 1
+            
+            else:
+                image_folder = "{}/{}/".format(args.image_dir, label_name.split(".")[0])
+                for frame in sorted(os.listdir(image_folder)):
+                    if int(frame[6:11]) >= frame_start and int(frame[6:11]) <= frame_end:
+                        im = Image.open(image_folder + frame)
+                        im.save('{}/{}/{}/{}.jpg'.format(args.data_out_path, args.phase, label_folder, frame[6:11].lstrip('0')))
+
+        if args.phase == "train":
+            train_df = pd.DataFrame(label_anns)
+            train_meta_df = pd.DataFrame(meta_anns)
+            train_df.to_feather(args.label_out_path + "labels_train_{}.feather".format(i))
+            train_meta_df.to_feather(args.label_out_path + "meta_train_{}.feather".format(i))
+        else:
+            test_df = pd.DataFrame(label_anns)
+            test_meta_df = pd.DataFrame(meta_anns)
+
+            test_df.to_feather(args.label_out_path + "labels_test_{}.feather".format(i)) 
+            test_meta_df.to_feather(args.label_out_path + "meta_test_{}.feather".format(i)) 
+
+    files = ["labels_test", "meta_test", "labels_train", "meta_train"]
+    for file in files:
+        final_df = pd.DataFrame()
+        for i in len(os.listdir(args.label_folder)):
+            temp_df = pd.read_feather(args.label_out_path + "{}_{}.feather".format(file, i))
+            final_df = pd.concat([final_df, temp_df]).reset_index(drop=True, inplace=False)
+        print(final_df)
+        final_df.to_feather(args.label_out_path + "{}.feather".format(file))
+        
+
+
+if __name__ == "__main__":
+    ap = argparse.ArgumentParser()
```

Comment:
Let's add some description to this tool so that we know what it is for and how to use it when we come back to it (module doc-string, CLI arg parser description, argument help docs, etc.)